script.js

```diff
@@ -1,4 +1,145 @@
+// Basic Scissors Paper Stone
+
+// Create a basic version of Scissors Paper Stone where the user inputs one of "scissors", "paper", or "stone", the program internally randomly chooses scissors, paper, or stone, and the program outputs whether the user won, the program won, or it's a draw.
+
+// Rules: scissors beats paper, paper beats stone, and stone beats scissors. If both parties choose the same object, it's a draw.
+// Input Validation
+
+// Sometimes the user types bad and types something other than "scissors", "paper", or "stone" during gameplay. Add input validation to kindly let the user know that there are only 3 input options, and ask them to try again.
+
+// user inputs choice
+// default answer is lose
+// if wrong input, prompt user to try again
+// program generates rsp
+// program compares generated rsp to choice
+// program outputs three outcomes - win or draw or lose
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  userInput = input;
+  console.log("input: " + input);
+  var gameResult = aiResult();
+  console.log("result: " + gameResult);
+  console.log("result format: " + rspFormat);
+
+  var newFormat = format();
+  console.log("format: " + newFormat);
+
+  //program compares generated rsp to choice
+  //draw condition
+  if (
+    (userInput == "scissors" && gameResult == "scissors") ||
+    (userInput == "paper" && gameResult == "paper") ||
+    (userInput == "stone" && gameResult == "stone")
```

Comment:
There is something know as [SCREAM_CASE constants ](https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/)

It's another way to name variables that remain unchanged through out the program eg.
```
var STONE = "stone"
var SCISSORS = "scissors"
var PAPER = "paper"
```